### PR TITLE
Feature/renew canal

### PR DESCRIPTION
**Pull request by takama**

### CODE REVIEW

File: roles/canal/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# Executable files path
+k8s_bin_dir: /usr/bin
+
+# Master hosts nsmes
+k8s_master_hosts: {}
+
+# Nodes hosts names
+k8s_node_hosts: {}
+
+# Hyperkube image
```
Comment (STASiAN) [inline roles/canal/defaults/main.yml]: is it hyperkube necessary here!?

File: roles/canal/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
+Ansible Role: Canal
+===================
+
+This role install Canal Network Plugin on Redhat linux based systems.
```
Comment (STASiAN) [inline roles/canal/README.md]: correct spelling for redhat looks like Red Hat

Comment (takama) [reply roles/canal/README.md]: ok, fixed

File: roles/canal/defaults/main.yml (hunk 1 above)
Comment (takama) [reply roles/canal/defaults/main.yml]: fixed

**Review by STASiAN — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]

**Pull request merged**